src/main/java/io/openliberty/sample/application/CrewService.java

```diff
@@ -53,30 +70,85 @@ public String add(CrewMember crewMember) {
 			}
 			return messages.build().toString();
 		}
+	        
+			 Document newCrewMember = new Document();
+			 newCrewMember.put("Name",crewMember.getName());
+			 newCrewMember.put("Rank",crewMember.getRank());
+			 newCrewMember.put("CrewID",crewMember.getCrewID());
 
-        //TODO Insert CrewMember into cloudant database
-		return "";
+		     PostDocumentOptions createDocumentOptions =
+		            new PostDocumentOptions.Builder()
+		                .db(dbname)
+		                .document(newCrewMember)
+		                .build();
+
+		     DocumentResult createDocumentResponse = client
+		            .postDocument(createDocumentOptions)
+		            .execute()
+		            .getResult();
+
+		return createDocumentResponse.toString();
```

Comment:
If I remember correctly, the application is expecting an empty string for success, otherwise it will interpret the response as an error. Unless DocumentResult functions that way then this will need to be changed